### ISSUE TITLE
fileUpload returns guzzle object

### ISSUE THREAD

**Issue by monove** [cartalyst/stripe#62]
When doing a call like this
```
$response = $stripe->fileUploads()->create($file,'identity_document');
```
`$response` is a guzzle object instead of the fileUpload object.

Currently, in order to get the contents I must do
```
json_decode($response->getBody()->getContents());
```

**Comment by brunogaspar** [comment]
Want to send a PR to the `1.0` branch?

**Comment by brunogaspar** [comment]
This is now fixed on 81144a2148ef64e6423cbbe74b4efe5c9cf2fbaf
